lib/cog/pipeline/util.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Cog.Pipeline.Util do
+
+  alias Cog.Pipeline.Destination
+
+  def here_destination(request) do
```

Comment:
We don't need the output classification logic anymore with the new pipeline structure, correct?

The name is slightly misleading, given that we're actually returning a list (even though it only has one item).

Could this just be a function in `Cog.Pipeline.Destination` directly?